doc/math_engine/katex.page

```diff
@@ -5,10 +5,24 @@ title: KaTeX
 ## Math Engine KaTeX
 
 This math engine uses [KaTeX] via the [katex rubygem][katex-gem] to convert TeX math formulas into
-KaTeX HTML (using [ExecJS] under the hood).
+KaTeX HTML, using [ExecJS] under the hood. This eliminates the need for client-side math-rendering
+Javascript. This engine is somewhat similar to the [SsKaTeX] math engine, but easier to use and
+suitable for kramdown input from untrusted users.
 
-This eliminates the need for client-side math-rendering JavaScript. Your HTML templates still need
-to include the KaTeX CSS (see the [gem's documentation][katex-gem#assets] on this).
+Requirements for running kramdown with math engine KaTeX:
+
+- Ruby gem [`katex`][katex-gem],
+- Ruby gem [`execjs`][ExecJS],
```

Comment:
This is a `runtime_dependency` of the gem so there is no need to install it separately:
https://github.com/glebm/katex-ruby/blob/e432853a5ae1122e8dc8ce96147bded41494baef/katex.gemspec#L25

Reply:
That's just saying that the requirement usually gets fulfilled by dependency resolvers, but it's still a requirement. And it informs about what will (have to) be found in a functioning installation.